dlk/hw/make_os/build/make_rootfs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+	echo "Error: Number of argument should be 1"
+	exit 0
+fi
+
+BIT_32_OR_64=$1
+
+if [ $1 = 32 ]; then
+	DEBOOTSTRAP_ARCH=armhf
+	QEMU_ARCH=arm
+	OUTPUT_FNAME=rootfs_armhf.tgz
+elif [ $1 = 64 ]; then
+	DEBOOTSTRAP_ARCH=arm64
+	QEMU_ARCH=aarch64
+	OUTPUT_FNAME=rootfs_arm64.tgz
+else
+	echo "Error: Argumnet should be 32 or 64"
+	exit 1
+fi
+
+BUILD_DIR=/build
+ROOTFS_DIR=$BUILD_DIR/rootfs
+
+# Chcek if the $ROOTFS_DIR exists or not
+if [ -d $ROOTFS_DIR ]; then
+	echo "Error: $ROOTFS_DIR already exists"
+	exit 1
+fi
+
+# Check if $ROOTFS_DIR/$OUTPUT_NAME exists or not
+if [ -e $BUILD_DIR/$OUTPUT_FNAME ]; then
+	echo "Error: $BUILD_DIR/$OUTPUT_FNAME already exists"
+	exit 1
+fi
+
+# Make rootfs
+debootstrap --arch=$DEBOOTSTRAP_ARCH --keyring=/usr/share/keyrings/ubuntu-archive-keyring.gpg --verbose --foreign bionic $ROOTFS_DIR
+cp /usr/bin/qemu-${QEMU_ARCH}-static $ROOTFS_DIR/usr/bin/
```

Comment:
debootstrap and copy qemu-*-static can be replaced by `qemu-debootstrap`.

Reply:
@primenumber 
Thank you for helpful comment.
I looked into `qemu-debootstrap` by your comment and I found that it might be able to make code shorter. But `qemu-debootstrap` is just the wrapper of `debootstrap` and `qemu-*-static`.
I think I want to use `debootstrap` and `qemu-*-static` instead of `qemu-bootstrap` this time because it's easy to understand what commands acually do more precisely.
How do you think about this ?

Reply:
Hmm, you have a point.